segment_sc_contrast-agnostic/README.md

```diff
@@ -0,0 +1,37 @@
+Segment SC using contrast-agnostic MONAI model from PSIR contrast and perform vertebral labeling
+
+1. Create a conda virtual environment and install dependencies
+
+```console
+conda create -n monai python=3.8
```

Comment:
TODO: Update to `python=3.9`
Context: https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/60#issuecomment-1753109865

Reply:
Solved in https://github.com/ivadomed/canproco/pull/44/commits/688d1a0f99f98171825644da4bf5d5c972be38c6